server/jetstream_api.go

```diff
@@ -146,6 +146,21 @@ const (
 	// JetStreamAdvisoryConsumerMaxDeliveryExceedPre is a notification published when a message exceeds its delivery threshold.
 	JSAdvisoryConsumerMaxDeliveryExceedPre = "$JS.EVENT.ADVISORY.CONSUMER.MAX_DELIVERIES"
 
+	// JSAdvisoryStreamCreatedPre notification that a stream was created
+	JSAdvisoryStreamCreatedPre = "$JS.EVENT.ADVISORY.STREAM.CREATED"
+
+	// JSAdvisoryStreamDeletedPre notification that a stream was deleted
+	JSAdvisoryStreamDeletedPre = "$JS.EVENT.ADVISORY.STREAM.DELETED"
+
+	// JSAdvisoryStreamUpdatedPre notification that a stream was updated
+	JSAdvisoryStreamUpdatedPre = "$JS.EVENT.ADVISORY.STREAM.UPDATED"
+
+	// JSAdvisoryConsumerCreatedPre notification that a template created
+	JSAdvisoryConsumerCreatedPre = "$JS.EVENT.ADVISORY.CONSUMER.CREATED"
+
+	// JSAdvisoryConsumerDeletedPre notification that a template deleted
+	JSAdvisoryConsumerDeletedPre = "$JS.EVENT.ADVISORY.CONSUMER.DELETED"
```

Comment:
I went with the `xPre` pattern we used here for other advisories where we're also just doing string concats